<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTeam</class>
 <widget class="QDialog" name="addTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Team Quick</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="LabelName">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LabelLocation">
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="LabelTier">
      <property name="text">
       <string>Tier:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editTier"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editLocation"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editName"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="buttonAdd">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
